.storybook/components/DesignTokens/Tier1/FontSizes.jsx

```diff
@@ -1,30 +0,0 @@
-import PropTypes from 'prop-types';
```

Comment:
Were these ones being used b/c they aren't used anywhere?

Reply:
That's right, I can't find any usages of them anywhere.

Looking at c8a3bbc28fb688c94f99b0ff73a64bc4cda0b8d8, these were probably part of Big Medium's template.

Reply:
I went back in time for a while and don't think either of these were being used